.vib/consul/goss/goss.yaml

```diff
@@ -0,0 +1,44 @@
+port:
+  tcp6:8400:
+    listening: true
+    ip:
+    - '::'
+file:   
+  /opt/bitnami/consul/conf/consul.json:
+    mode: "0644"
+    filetype: file
+    contains: 
+      - "8500"
+      - "8600"
+      - "8400"
+      - "8301"
+    exists: true
+command:
+  user-id-test:
+    exec: if [ "$(id -u)" -eq 0 ]; then exit 1; fi
+    exit-status: 0
+    stdout: []
+    stderr: []
+  check-env:
+    exec: env
+    exit-status: 0
+    stdout: 
+      - "HOSTNAME=consul-0"
+      - "CONSUL_DATACENTER=datacenter1"
+    stderr: [] 
```

Comment:
I would like to hear your opinion on this, but to me, usually there is no such a big return in checking environment variables are set: they are just a way to pass down information to our configuration scripts.

Instead, if we want to assert that `datacenter1` is effectively the name of the datacenter I would resort to other means when possible.

TL;DR
An environment variable having a specific value does not guarantee the logic has correctly applied it.

Reply:
I see your point, I've replaced this test with one that seems to be more useful.